app/templates/completed.js

```diff
@@ -0,0 +1,31 @@
+const html = require('choo/html');
+const progress = require('./progress');
+const { fadeOut } = require('../utils');
+
+module.exports = function(state, emit) {
+  const div = html`
+  <div id="download" class="fadeIn">
+    <div id="download-progress">
+      <div id="dl-title" class="title">${state.translate(
+        'downloadFinish'
+      )}</div>
```

Comment:
I'm a bit confused on what the difference between `state.translate('blah')` and just setting a `data-l10n-id="baz"` attribute and let l20n handle translations.

Reply:
The l20n lib is gone. We don't want two different libs both making changes to the same dom nodes. And bonus, this lets us do l10n on the server as well.

Reply:
ah, OK... in that case, we might have 2 lingering references to `data-l10n-id`:

```sh
$ git grep data-l10n-id

app/templates/error.js:    <img id="upload-error-img" data-l10n-id="errorAltText" src="${assets.get(
app/templates/notFound.js:        )}" id="expired-img" data-l10n-id="linkExpiredAlt"/>
```